src/ui/components/results/verticalresultscomponent.js

```diff
@@ -241,7 +242,24 @@ export default class VerticalResultsComponent extends Component {
   getVerticalURL (data = {}) {
     const verticalConfig = this._verticalsConfig.find(config => config.verticalKey === this.verticalKey) || {};
     const verticalURL = this._config.verticalURL || verticalConfig.url || data.verticalURL || this.verticalKey + '.html';
-    return addParamsToUrl(verticalURL, { query: this.query });
+
+    let tabOrder = getDefaultTabOrder(this._verticalsConfig, getUrlParams());
```

Comment:
Could we try to generalize all of this this logic into a helper method that generates the tabOrder param? We could then add it to verticalURL with addParamsToUrl (like before). Ideally, VerticalResults wouldn't contain any tabOrder logic. It also looks like we're not accounting for cases when verticalPages is specified in the nav component rather than in ANSWERS.init. It's very possible we don't need to handle this case, because it never worked before it's not a breaking change (and I think we've deprecated specifying verticalPages through the nav component anyways). I would double check with Rose that it's ok that we don't handle correctly merging the tabOrder with verticalPages if verticalPages is specified in the navigation component and not in ANSWERS.init.

Reply:
ok, sounds like we don't have to worry about doing weird magic in the sdk, but we will have to rejigger the theme to get the full benefits of your change. Still would prefer for this logic to be hidden from vertical results as much as possible, but think the overall idea is what we want!